src/builtins/systemio.rs

```diff
@@ -48,13 +69,12 @@ impl SystemIO {
         SystemIO::nothing()
             .allow_read()
             .allow_write()
-            .allow_open().yes_really()
+            .allow_open()
+            .yes_really()
```

Comment:
Or for example here where it hides the .yes_really() in with the other allow calls which I noticed in your original PR. There's really no way for the computer to know that this is "significant" here and that it shouldn't move it on to its own line.